waspc/examples/todoApp/sample.env.server

```diff
@@ -1,2 +1,3 @@
 TEST_ENV_VAR="I am test"
 DATABASE_URL=postgresql://postgres:postgres@localhost/todoapp
+#PORT=4041
```

Comment:
Maybe we can also set this to default value which is 3001.

Reply:
I'll adjust it.